### ISSUE TITLE
Compile DD with embree from deb package

### ISSUE THREAD

**Issue by shimwell** [pshriwise/double-down#20]
I've just tried rebuilding the openmc docker file using the deb embree package instead of the embree compile and I got as far as the DD compile.

I think DD might not be finding embree correctly but am not entirely sure what is happening.

Dockerfile is [here](https://github.com/Shimwell/openmc-1/blob/using_embree_deb_package/Dockerfile)

![dd_compile_error_with_deb_embree](https://user-images.githubusercontent.com/8583900/131124887-b55afa51-f7d2-4dfc-8952-59a7575c050f.png)

**Comment by ebknudsen** [comment]
Comment from left field:
I encountered the same problem when i tried using the .deb embree on my machines.
In my case it was caused by a couple of missing .cmake files. These are not present in the libembree-dev package from Ubuntu 20.04 focal fossa (https://packages.ubuntu.com/focal/amd64/libembree-dev/filelist). Good news is they are present in the libembree-dev package distributed with 22.04 jammy jellyfish (https://packages.ubuntu.com/jammy/amd64/libembree-dev/filelist)

So whenever ubuntu:latest starts pointing to 22.04  this should simply solve itself I believe.
